examples/js/index.js

```diff
@@ -71,6 +52,8 @@ const SlideFragments = () => (
   </>
 );
 
+const template = <DefaultTemplate />;
```

Comment:
This is not inline because the **one page** code is generated from this. The JSX → HTM transpiler does strange things with indentation when you have a render prop like `<A prop={<B />} />` so we just make it a var.